### PR TITLE
Did another pass on the block package

### PR DESCRIPTION
**Pull request by amusingimpala75**
Went through the blocks package and did some more mapping

### CODE REVIEW

File: mappings/net/minecraft/block/Dirt.mapping [hunk 1]
```diff
@@ -1 +1,6 @@
 CLASS net/minecraft/class_65 net/minecraft/block/Dirt
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Index is ignored by the parameter
```
Comment (sschr15) [inline mappings/net/minecraft/block/Dirt.mapping:3]: cool comments but eh

File: mappings/net/minecraft/block/Door.mapping [hunk 2]
```diff
@@ -1,3 +1,6 @@
 CLASS net/minecraft/class_49 net/minecraft/block/Door
+	COMMENT @param id Id is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Door.mapping:2]: same as dirt: great comment but eh

File: mappings/net/minecraft/block/Fire.mapping [hunk 3]
```diff
@@ -1,3 +1,34 @@
 CLASS net/minecraft/class_51 net/minecraft/block/Fire
-	METHOD method_131 (III)V
+	FIELD field_181 burnDestroyChangeIdMap [I
+	FIELD field_182 flamabilityIdMap [I
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Texture Index is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Fire.mapping:5]: same as above

File: mappings/net/minecraft/block/Ladder.mapping [hunk 4]
```diff
@@ -1 +1,6 @@
 CLASS net/minecraft/class_70 net/minecraft/block/Ladder
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Texture Index is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Ladder.mapping:3]: same as above

File: mappings/net/minecraft/block/Leaves.mapping [hunk 5]
```diff
@@ -1,2 +1,9 @@
 CLASS net/minecraft/class_68 net/minecraft/block/Leaves
+	FIELD field_192 myTextureIndex I
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Texture Index is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Leaves.mapping:4]: same as above

File: mappings/net/minecraft/block/Log.mapping [hunk 6]
```diff
@@ -1 +1,4 @@
 CLASS net/minecraft/class_42 net/minecraft/block/Log
+	COMMENT @param id Id is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Log.mapping:2]: same as above

File: mappings/net/minecraft/block/Obsidian.mapping [hunk 7]
```diff
@@ -1 +1,6 @@
 CLASS net/minecraft/class_74 net/minecraft/block/Obsidian
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Texture Index is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Obsidian.mapping:3]: same as above

File: mappings/net/minecraft/block/Rail.mapping [hunk 8]
```diff
@@ -1 +1,16 @@
 CLASS net/minecraft/class_66 net/minecraft/block/Rail
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Texture Index is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Rail.mapping:3]: same as above

File: mappings/net/minecraft/block/SignBlock.mapping [hunk 9]
```diff
@@ -1,3 +1,8 @@
 CLASS net/minecraft/class_50 net/minecraft/block/SignBlock
 	FIELD field_179 signId I
 	FIELD field_180 signClass Ljava/lang/Class;
+	COMMENT @param id Id is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/SignBlock.mapping:4]: here too

File: mappings/net/minecraft/block/Sponge.mapping [hunk 10]
```diff
@@ -1 +1,4 @@
 CLASS net/minecraft/class_48 net/minecraft/block/Sponge
+	COMMENT @param id Id is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Sponge.mapping:2]: and here

File: mappings/net/minecraft/block/StoneSlab.mapping [hunk 11]
```diff
@@ -1,2 +1,6 @@
 CLASS net/minecraft/class_53 net/minecraft/block/StoneSlab
 	FIELD field_184 notSlab Z
+	COMMENT @param id Id is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/StoneSlab.mapping:3]: here as well

File: mappings/net/minecraft/block/TNT.mapping [hunk 12]
```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/class_60 net/minecraft/block/TNT
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Texture Index is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/TNT.mapping:3]: also here

File: mappings/net/minecraft/block/Torch.mapping [hunk 13]
```diff
@@ -1 +1,11 @@
 CLASS net/minecraft/class_44 net/minecraft/block/Torch
+	COMMENT @param id Id is ignored by the constructor
+	COMMENT @param textureIndex Texture Index is ignored by the constructor
```
Comment (sschr15) [inline mappings/net/minecraft/block/Torch.mapping:3]: here as well part 2

**Review by sschr15 — COMMENTED** [review]
unnecessary comments because of proguard should maybe get disappeared

**Review by sschr15 — CHANGES_REQUESTED** [review]
PR blocked until this can be figured out
what is better name for what the method does and where it's called

image

**Comment by amusingimpala75** [comment]
Well, it makes sense to call it can connect towards, because sometimes it can connect towards an air block which has a rail above it. 
If the block xyz is a rail OR xyz + 1y is rail and xyz is block OR xyz - 1y is rail and xyz is air then it connects. xyz does not have to be a rail itself.

ἔρρωσο

**Pull request closed without merging**